javascript/src/main/pages/Admin/ViewLogins.js

```diff
@@ -25,10 +25,15 @@ const ViewLogins = () => {
   const { getAccessTokenSilently: getToken } = useAuth0();
   const { data:loginTable } = useSWR(["/api/admin/logins", getToken], fetchWithToken);
 
+  if (loginTable){
+    var size = loginTable.length;
+  }
+
```

Comment:
Is there a reason why `size` needs to be declared as an extra variable? Can we just access `loginTable.length` directly from the render code? IMO it looks nicer this way.

Reply:
loginTable is undefined in some instances, is there any way for my to use .length on an undefined object?

Reply:
No, but you can the check for whether it exists inline using ternary operators.
Alternatively, you could logical AND the `<p>` tag and `loginTable` to only show the "Number of logins:" text when the data is loaded.